e2e/rbd.go

```diff
@@ -370,6 +370,45 @@ var _ = Describe("RBD", func() {
 				})
 			}
 
+			By("Validate AnnotateFSResize", func() {
+				// this featuregate is available since kube v1.21.
+				expandSize := "3Gi"
+				pvc, err := loadPVC(pvcPath)
+				if err != nil {
+					e2elog.Failf("failed to load PVC: %v", err)
+				}
+				pvc.Namespace = f.UniqueName
+
+				err = createPVCAndvalidatePV(f.ClientSet, pvc, deployTimeout)
+				if err != nil {
+					e2elog.Failf("failed to create PVC: %v", err)
+				}
+				// validate created backend rbd images
+				validateRBDImageCount(f, 1, defaultRBDPool)
+
+				// resize PVC
+				err = expandPVCSize(f.ClientSet, pvc, expandSize, 1)
```

Comment:
why expand is failing here?

Reply:
```May 11 12:40:45.628: INFO: pvc state FileSystemResizePending
May 11 12:40:45.628: INFO: failed to expand PVC to size(3Gi): timed out waiting for the condition```

@Madhu-1

Reply:
@humblec it is not showing what is the exact reason for failure.